CITATION.cff

```diff
@@ -0,0 +1,110 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: Nex-generation file format (NGFF)
```

Comment:
typo `Nex-`
Should the title include `OME`?

Reply:
Typo fix pushed. Re: "OME" good question. It's hard to do the expansion, then. Maybe let's see how it looks when we start using it.